aws/cloudtrail/main.tf

```diff
@@ -5,21 +5,28 @@ resource "aws_cloudtrail" "mod" {
   enable_logging                = true
   enable_log_file_validation    = true
 
-  cloud_watch_logs_group_arn    = var.cloud_watch_logs_group_arn
-  cloud_watch_logs_role_arn     = var.cloud_watch_logs_role_arn
+  cloud_watch_logs_group_arn = var.cloud_watch_logs_group_arn
```

Comment:
Seems like the changes in this file are unrelated, and shouldn’t be a part of the PR?